crates/sui/src/client_commands.rs

```diff
@@ -945,12 +945,9 @@ impl SuiClientCommands {
                     .transaction_builder()
                     .transfer_sui(from, object_id, gas_budget, to, amount)
                     .await?;
-                let data1 = data.clone();
-                let intent_msg = IntentMessage::new(Intent::default(), data);
-                SuiClientCommandResult::SerializeTransferSui(
-                    Base64::encode(bcs::to_bytes(&intent_msg)?.as_slice()),
-                    Base64::encode(bcs::to_bytes(&data1).unwrap()),
-                )
+                SuiClientCommandResult::SerializeTransferSui(Base64::encode(
+                    bcs::to_bytes(&data).unwrap(),
+                ))
```

Comment:
not necessarily for this PR (as it didn't change behavior), but ideally let's document/justify all unwraps.